hw_16/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { customCalculator } from './custom-calculator';
+
+// positive tests
+customCalculator(1, 2, '+', 3);
+customCalculator(5, 2, '-', 3);
+customCalculator(4, 2, '/', 3);
+customCalculator(1, 3, '*', 3);
+customCalculator(6, 6, '*', 36);
+customCalculator(10, 9, '-', 1);
```

Comment:
Please write your tests using test design technics.
For example we should check two positive numbers multiplication, multiplication of negative and positive number, multiplication with `0`, etc.